Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -1382,6 +1534,8 @@ - (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceO
 }
 
 - (void)scrollToElementOnScreenBeforeRotate {
+    // FIXME: rotating portrait/landscape repeatedly causes the webview to scroll down instead of maintaining the same position
+    return;
```

Comment:
Do we just need to adjust finalScrollOffset by the header height or is it more complicated?